.changelog/17731.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:breaking-change
+connect: Disable peering by default in connect proxies for Consul 1.13. This change was made to prevent inefficient polling
+queries from having a negative impact on server performance. Peering in Consul 1.13 is an experimental feature and is not
+recommended for use in production environments. If you still wish to use the experimental peering feature, ensure
+[`peering.enabled = true`](https://developer.hashicorp.com/consul/docs/v1.13.x/agent/config/config-files#peering_enabled)
+is set on all agents and servers.
```

Comment:
nit: I think it's either "all clients and servers" or just "all agents". I think the former is more clear.